kasa/klaptransport.py

```diff
@@ -271,14 +276,18 @@ async def perform_handshake(self) -> Any:
         self._session_cookie = None
 
         local_seed, remote_seed, auth_hash = await self.perform_handshake1()
-        if cookie := self._http_client.get_cookie(  # type: ignore
-            self.SESSION_COOKIE_NAME
-        ):
+        http_client = self._http_client
+        if cookie := http_client.get_cookie(self.SESSION_COOKIE_NAME):  # type: ignore
             self._session_cookie = {self.SESSION_COOKIE_NAME: cookie}
         # The device returns a TIMEOUT cookie on handshake1 which
         # it doesn't like to get back so we store the one we want
```

Comment:
This comment implies its not always there so I added fallback to the const if its missing